src/github.rs

```diff
@@ -208,49 +210,69 @@ pub async fn command_fetch_release_distributions(args: &ArgMatches) -> Result<()
 
             let name = zf.name().to_string();
 
+            let parts = name.split('-').collect::<Vec<_>>();
+
+            if parts[0] != "cpython" {
+                println!("ignoring {} not a cpython artifact", name);
+                continue;
+            };
```

Comment:
We later filter these out https://github.com/indygreg/python-build-standalone/blob/68a3e155a00835c5b8c5c2256eff5beaf65ab447/src/github.rs#L365 so we can avoid downloading them here to save disk space